components/build/tekton-chains/chains-certs.yaml

```diff
@@ -113,3 +113,38 @@ spec:
       terminationGracePeriodSeconds: 30
       serviceAccount: chains-secrets-admin
       serviceAccountName: chains-secrets-admin
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: tekton-chains-signing-secret
+  namespace: tekton-chains
+  annotations:
+    argocd.argoproj.io/hook: Sync
+    argocd.argoproj.io/hook-delete-policy: HookSucceeded
```

Comment:
Please remove this annotation, it's deleting the job, it's better to use default delete policy which deletes job before running new one. So it's better for debugging.

Also I noticed that when I remove the secret and run secret then it job fails for first time on:
```
I0325 13:02:26.662958 8 request.go:665] Waited for 1.160947575s due to client-side throttling, not priority and fairness, request: GET:https://172.30.0.1:443/apis/apps/v1?timeout=32s
I0325 13:02:36.861548 8 request.go:665] Waited for 11.35708957s due to client-side throttling, not priority and fairness, request: GET:https://172.30.0.1:443/apis/console.openshift.io/v1alpha1?timeout=32s
Successfully created secret signing-secrets in namespace tekton-chains
Error: open cosign.pub: permission denied
main.go:46: error during command execution: open cosign.pub: permission denied
```
But then it's restarted and fixed.

Reply:
It will be issue of the current working directory, probably should be fixed by `cd /tmp/` bofore running cosign command (current working dir is read-only, so it failed on saving cosign.pub, which is fine but better to not end with fail).